.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Stale PR tracking
+
+on:
+  schedule:
+    - cron: '0 9 * * 1-5' # 9am on weekdays
+
+permissions:
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          stale-pr-message: 'This pull request has gone a while without any activity. Tagging in the Trino developer relations team - @bitsondatadev @colebow @mosabua'
```

Comment:
Starting the sentence with "Tagging in the ..." made it sound weird when I first read it. Maybe "Tagging the ..." would sound better?

Nit: I'd also change the dash to a colon